pkgs/development/python-modules/typing-extensions/default.nix

```
@@ -8,15 +8,15 @@
 
 buildPythonPackage rec {
   pname = "typing-extensions";
-  version = "4.5.0";
+  version = "4.7.1";
   format = "pyproject";
 
   disabled = pythonOlder "3.6";
```

```suggestion
  disabled = pythonOlder "3.7";
```

see https://github.com/python/typing_extensions/blob/4.7.1/pyproject.toml#L12

---

Done in [force-push](https://github.com/NixOS/nixpkgs/compare/7fc3327aecfd10adc917b265e23e3d0b12d2cf90..ebccac00dbf2301b08cc5aa67504e65743b24325).

Maybe worth a headsup that `4.7.*` will also be the last version to support `3.7` moving on to `3.8`-only afterwards.